addons/nuxeo-core-binarymanager-cloud/nuxeo-core-binarymanager-s3/src/main/java/org/nuxeo/ecm/blob/s3/S3BlobProvider.java

```diff
@@ -237,13 +237,11 @@ public BlobStatus getStatus(ManagedBlob blob) throws IOException {
         if (StorageClass.Standard.toString().equals(storageClass)) {
             storageClass = null;
         }
-        // x-amz-restore absent
-        // x-amz-restore: ongoing-request="true"
-        // x-amz-restore: ongoing-request="false", expiry-date="Fri, 23 Dec 2012 00:00:00 GMT"
```

Comment:
Please keep the comments, they're important for illustration of the possible cases. You may add an additional one on top saying `// the object has either standard storage class, or in the glacier storage class can have one of these 3 states:`